templates/template-spring-mvc-maven-mongodb/files/README.md

```diff
@@ -0,0 +1,19 @@
+# Spring MVC Maven template for Platform.sh
+
+This project provides a starter kit for Spring Boot Maven with MySQL projects hosted on [Platform.sh](http://platform.sh).
```

Comment:
I missed this in the last PR, but these READMEs are identical. This doesn't address MVC and it still says MySQL instead of Mongo.